libs/ui/search/src/lib/results-hits-search-kind/results-hits-search-kind.component.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  SimpleChanges,
+  Output,
+} from '@angular/core'
+import { Choice } from '@geonetwork-ui/ui/inputs'
+
+@Component({
```

Comment:
Should be standalone, I think.

Reply:
Was this suggestion linked to the npm package generation error? it was a missing import in the index file for the module actually. I did not add Standalone because none of the components in this lib are standalone.

Reply:
Indeed, the error pointed my attention to this, although both things are independent. I think the components in the lib are not standalone, because they have not been "migrated" yet. If I remember right, the idea was to make all components standalone. So if you think it's not much work, I'd go for standalone for the new component, otherwise we'll take care of it when making the entire lib standalone.

Reply:
Ok i wasn't aware of that topic. I tried to switch to standalone, but then i had too much modifications to make it work (in UTs, but also in module declaration and also in the parent container component). I hope it is ok for you if we keep this topic for another time as this is not the subject of this PR and i fear this update is not straightforward.

Reply:
Yes, of course, no problem. Thanks for checking, anyway!